constants_test.py

```diff
@@ -44,3 +45,10 @@ def test_constants_and_feconf_are_consistent(self):
         self.assertIn(
             feconf.SYSTEM_COMMITTER_ID, constants.constants.SYSTEM_USER_IDS)
         self.assertEqual(len(constants.constants.SYSTEM_USER_IDS), 2)
+
+    def test_all_comments_are_removed_from_json_text(self):
+        """Tests if comments are removed from json text."""
+        with open(os.path.join('assets', 'constants.js'), 'r') as f:
```

Comment:
Don't do this -- the file will change, and because of that, the test is quite weak.

Make a dummy simple test file in core/tests/data instead. Then assert that its contents after the operation are what you expect (compare it to a "golden string").

Reply:
Done.